tests/post_training/pipelines/base.py

```diff
@@ -418,6 +418,9 @@ def save_compressed_model(self) -> None:
             ov.serialize(ov_model, self.path_compressed_ir)
         elif self.backend in OV_BACKENDS:
             self.path_compressed_ir = self.output_model_dir / "model.xml"
+            from openvino._offline_transformations import apply_moc_transformations
+
+            apply_moc_transformations(self.compressed_model, cf=True)
```

Comment:
Could you please explain the reason for using this transformation here?

Reply:
This is done to align amount of FQ between OV and FX backends for the VIT model 
<img width="816" alt="image" src="https://github.com/user-attachments/assets/1df98379-11c3-4a2b-9744-3f45672ab0d6">